programs/mango-v4/src/accounts_ix/openbook_v2_settle_funds.rs

```diff
@@ -73,4 +73,5 @@ pub struct OpenbookV2SettleFunds<'info> {
     pub base_oracle: UncheckedAccount<'info>,
```

Comment:
the comment on these oracles I don't get "in the parent"? Probably a c&p error. Also, I didn't find validation, please add!